src/helpers/embedding_helpers.py

```diff
@@ -0,0 +1,24 @@
+from openai import AzureOpenAI
+from typing import List
+import config as cfg
+
+class OpenAIEmbeddingFunction:
```

Comment:
This is nice! We should move all LLM stuffs exactly in a single separate helper like that! :)